src/fidesops/ops/email_templates/get_email_template.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from jinja2 import Environment, FileSystemLoader, Template, select_autoescape
+
+from fidesops.ops.common_exceptions import EmailTemplateUnhandledActionType
+from fidesops.ops.email_templates.template_names import (
+    SUBJECT_IDENTITY_VERIFICATION_TEMPLATE,
+)
+from fidesops.ops.schemas.email.email import EmailActionType
+
+logger = logging.getLogger(__name__)
+
+template_env = Environment(
+    loader=FileSystemLoader("templates"),
```

Comment:
The jinja docs says it will take a relative path for the templates, but it seems unhappy about the update. I'm looking to see if I can find a solution.

Reply:
I noticed that too. I just pushed up an update that might work. It builds an absolute path based off of the file the code is ran from.

Reply:
Looks like a good solution

Reply:
The most recent run failed. It worked locally before I pushed it up. I'll double check that it's working as expected on my local machine